ktor-server/ktor-server-config-yaml/jvmAndNix/test/YamlConfigTest.kt

```diff
@@ -133,6 +133,18 @@ class YamlConfigTest {
         }
     }
 
+    @Test
```

Comment:
Please add a test when there is a default value, and an environment variable is set.